data/json/items/comestibles/brewing.json

```diff
@@ -44,6 +44,51 @@
     "flags": [ "NUTRIENT_OVERRIDE" ],
     "comestible_type": "DRINK"
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "brew_gin",
+    "name": "gin mash",
```

Comment:
It's the same name as in `gin_mash`, is this intended?

Reply:
> It's the same name as in `gin_mash`, is this intended?

When you mix botanicals with the other alcohol it creates a 'mash' for gin, so it is intended in this case. After the mash is completed (the flavours of the botanicals have had time to be added) it is still gin mash. The descriptions should help identify the two.

Reply:
Necessity to read description to tell apart two different items with identical names is a bad practice. Please differentiate names somehow.

Reply:
> Necessity to read description to tell apart two different items with identical names is a bad practice. Please differentiate names somehow.

Guess I could just add 'unfinished' to the first part.